pkg/reconcile/pipeline/context/service.go

```diff
@@ -49,6 +50,17 @@ type service struct {
 	bindingDefinitions    []binding.Definition
 }
 
+func (s *service) IsDirectSecretReference() bool {
+	if s.serviceRef.Kind == "Secret" && s.serviceRef.Version == "v1" {
```

Comment:
group should be empty

Reply:
Done. 1bf562899688f37d3b28faaa2b8ca8c680441719

Reply:
I had to change commit because if DCO.  This is the new commit: 273c29d7b6ae20431f7c54d534e4ecddcdb16b8e